README.md

```diff
@@ -956,6 +956,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 * [chronotope/chrono](https://github.com/chronotope/chrono) — [<img src="https://api.travis-ci.org/chronotope/chrono.svg?branch=master">](https://travis-ci.org/chronotope/chrono)
 * [yaa110/rust-persian-calendar](https://github.com/yaa110/rust-persian-calendar) — [<img src="https://api.travis-ci.org/yaa110/rust-persian-calendar.svg?branch=master">](https://travis-ci.org/yaa110/rust-persian-calendar)
+* [Mnwa/ms](https://github.com/Mnwa/ms) — [<img src="https://github.com/Mnwa/ms/workflows/build/badge.svg?branch=master">](https://github.com/Mnwa/ms/actions?query=workflow%3Abuild)
```

Comment:
Can you please add a description for the library? I know there's none for the ones above that, but that's a legacy problem we need to fix and new additions should have descriptions.

Reply:
No problem

Reply:
@palfrey done